cmd/bootstrapper/server.go

```diff
@@ -134,8 +134,40 @@ func (s *Server) Start(ctx context.Context, errCh chan error, params *NetworkPar
 		// start generating fallback data
 		s.eg.Go(
 			func() error {
-				s.genDataLoop(ctx, errCh, last, params.updateActiveSetTime, s.GenFallbackActiveSet)
```

Comment:
Would it make sense to move the retry logic you added here into the `genDataLoop` method?

Reply:
I actually really aren't a fan of this `genDataLoop`, and the other loop does not need a retry logic. That's why I inlined it (and I'd also tend to get rid of `getDataLoop` and inline it here too)

Reply:
Inlining is fine too, `getDataLoop` is now only used in one place anyway.